### DISCUSSION TITLE
Setting a gitignore personally

### DISCUSSION THREAD

**Discussion by kangz12345** [snu-quiqcl/qiwis#152] [Helpful references]
https://dev.to/fronkan/a-personal-gitignore-even-for-a-single-repository-4o7h

Following the instructions in the above article, you can set your own gitignore configuration.
For example, in macOS, `.DS_Store` files are created for many directories, which I want to ignore in git while it should not be in the repository `.gitignore`.

**Comment by BECATRUE** [comment]
Wow! Thanks for the reference.
